### DISCUSSION TITLE
How to use localize-content with JS?

### DISCUSSION THREAD

**Discussion by Sayed-Mahmoud** [LazZiya/XLocalizer#31] [Q&A] [answered]
an example: 
`var button = document.createElement('a');
                button.appendChild(document.createTextNode('Delete'));
                button.setAttribute('onclick', 'removeRow(this)');
                button.setAttribute('class', 'btn btn-danger');
                td.appendChild(button);`

**Comment by LazZiya** [comment, marked as answer]
Create a hidden HTML item with the localized text, then grab its value with js.

````html
<i style="display:none;" id="txt" localize-content>Delete</i>

<script>
    var delText = document.getElementById("txt").innerText;
    var button = document.createElement('a'); 
    button.appendChild(document.createTextNode(delText));
    ...
</script>
````
